File: examples/triangle/triangle.js
function setup(c) {
  c.cellModeCustom(" ");
}

function draw(c) {
  c.clear();
  c.stroke(".", "#ffffff", "#000000");
  c.fill("triangle", "#ffffff", "#000000");
  c.translate(c.width / 2, c.height / 2);
  c.rotate(c.framecount * 0.008);
  c.scale(1);
  c.triangle(5, 5, 15, 15, 2, 15);
}

File: examples/triangle/triangle.go
package main

import (
	"context"

	"github.com/emprcl/runal"
)

func main() {
	runal.Run(context.Background(), setup, draw)
}

func setup(c *runal.Canvas) {
	c.CellPadding(" ")
}

func draw(c *runal.Canvas) {
	c.Clear()
	c.Stroke(".", "#ffffff", "#000000")
	c.Fill("triangle", "#ffffff", "#000000")
	c.Translate(c.Width/2, c.Height/2)
	c.Rotate(float64(c.Framecount) * 0.008)
	c.Scale(1)
	c.Triangle(5, 5, 15, 15, 2, 15)
}
